hw/top_earlgrey/top_earlgrey_asic.core

```diff
@@ -9,8 +9,8 @@ filesets:
     depend:
       - lowrisc:systems:top_earlgrey:0.1
       - lowrisc:systems:top_earlgrey_pkg
-      - "!fileset_partner ? (lowrisc:systems:ast)"
-      - "fileset_partner ? (partner:systems:ast)"
+      - "!fileset_vendor ? (lowrisc:systems:ast)"
+      - "fileset_vendor ? (partner:systems:ast)"
```

Comment:
should the core file itself also be renamed to `vendor:...`?

Reply:
@msf can I ask why vendor instead of partner ? We all agreed on the name before adding the flags to the core files and also to the release script in foundry.

Reply:
The first component of the core name is meant to be the company producing this IP block, so it should actually be nuvoton. 

So if you really want to keep it "anonymous", something like lowrisc_partner (or partner, if it must be) would do the trick.

Vendor isn't a fantastic choice as it collides with our use of "vendoring" (e.g. as in those vendor directories we have in the tree.)

Reply:
My original suggestion was using fileset_nuvoton, but @tjaychen asked to keep it out of the opensource code. We all agreed on "partner". Not sure aht is the rush change towards vendor. @msf ???

Reply:
I am fine with either, as long as we stick to one. 

In this closed source [PR](https://opentitan-review.googlesource.com/c/asic/+/5440
), all DV sources are prefixed with `vendor_`. We should change that too in that case.

Reply:
Well, Since we (Nuvoton) would like to think of ourselves as partners more than a vendor, and also because of Phillips point above, I think we should keep "partner" as we all agreed few weeks ago.

thanks

Reply:
Oh I was only suggesting that for consistency across files. 
As @sriyerg has mentioned, the Nuvoton DV files use `vendor` as prefixes.
I am fine with any of the two, as long as its used consistently.

Sorry about the confusion.

Reply:
Reverted :)